corehq/apps/reports/standard/cases/basic.py

```diff
@@ -100,7 +101,6 @@ def _build_query(self):
         return query
 
     @property
-    @memoized
```

Comment:
@AmitPhulera was this removed for testing and now should be put back?

Reply:
Oh yes 🤦 
Thanks for catching that 😄

Reply:
Done.